inc/ScaffoldPackageCommand.php

```diff
@@ -273,7 +273,7 @@ public function package_readme( $args, $assoc_args ) {
 				$readme_args['commands'][] = array(
 					'name' => "wp {$command}",
 					'shortdesc' => $parent_command['description'],
-					'synopsis' => "wp {$command} {$parent_command['synopsis']}",
+					'synopsis' => "wp {$command}" . ( empty( $parent_command['subcommands'] ) ? " {$parent_command['synopsis']}" : "" ),
```

Comment:
@danielbachhuber Actually, I'm not exactly sure how to test this, or if it should be tested at all. The difference lies in commands running in the namespace, and subcommands. To test this you'd need two commands e.g. `brew` and `brew this`. This "bug" results in a space after the namespace command synopsis, like [here](https://github.com/wp-cli/admin-command#using).